src/apulse-stream.c

```diff
@@ -147,6 +147,11 @@ data_available_for_stream(pa_mainloop_api *a, pa_io_event *ioe, int fd,
                     "SND_PCM_STATE_DISCONNECTED state. Giving up.",
                     s->name ? s->name : "", s->c->name ? s->c->name : "");
                 break;
+            case SND_PCM_STATE_PRIVATE1:
```

Comment:
I think it's called "private" for a reason. Isn't it better to add `default` clause here?

Reply:
Sure, also fine! I'll update the PR in a minute.